runtime/lua/vim/lsp/client.lua

```diff
@@ -671,6 +671,11 @@ function Client:_request(method, params, handler, bufnr)
   bufnr = resolve_bufnr(bufnr)
   log.debug(self._log_prefix, 'client.request', self.id, method, params, handler, bufnr)
   local success, request_id = self.rpc.request(method, params, function(err, result)
+      -- Per LSP, don't show ContentModified error to the user.
+    if err and err.code ~= lsp.protocol.ErrorCodes.ContentModified then
+      vim.notify(string.format('[LSP][%s] %s: %s', self.name, err.code, err.message), vim.log.levels.ERROR)
```

Comment:
This might be a bit aggressive, but we seem to emit the error everywhere else anyway.